static/javascripts/components/survey_card.jsx

```diff
@@ -206,39 +226,62 @@ var Question = React.createClass({
 });
 
 
+
+
 var MQuestions = React.createClass({
+//set initial value
+getInitialState: function() {
+    return {value: 'select'};
+},
+//set value change
+changeHandler: function(event) {
+    this.setState({value: event.target.value});
+},
     render: function(){
       return (
-      <ul className="mdl-list">
-      <li>
-      <input type="text"  onChange={this.changeHandler}/>
-      </li>
-      <li>
-      <input type="text"  onChange={this.changeHandler}/>
-      </li>
-
-        </ul>
+      <div>
+      <form>
+          <div className="mdl-textfield mdl-js-textfield">
+            <p>Enter an option: &nbsp;<input className="mdl-textfield__input" type="text" id="questin_1" onChange={this.Handler}/></p>
+          </div>
+      </form>
+      <form>
+          <div className="mdl-textfield mdl-js-textfield">
+            <p>Enter an option: &nbsp;<input className="mdl-textfield__input" type="text" id="questin_1" onChange={this.Handler}/></p>
+          </div>
+      </form>
+      </div>
+
       );
     }
 });
 
 
+
+
+
+
 var SQuestions = React.createClass({
+
     render: function(){
       return (
-      <ul className="mdl-list">
```

Comment:
problem with mdl and React, may be: [http://stackoverflow.com/questions/31709013/material-design-lite-react-issues-with-tooltips](url)